### DISCUSSION TITLE
Poor performance on mobile with transparent canvas

### DISCUSSION THREAD

**Discussion by NeoHoffa** [creativelifeform/three-nebula#139] [Q&A] [answered]
I'm trying to run Nebula with my 8th Wall/THREE.js mobile experience. The 3D scene is rendered over top of the mobile camera feed, so the THREE scene/canvas is transparent. When I run the example below, the particles look great, but the scene's transparent background gets all ghosted and turns white. Looks fine on desktop, but not in Safari on iOS. 

https://codesandbox.io/s/three-nebula-quickstart-kz6uv?file=/src/index.js:281-297

Any thoughts?
Thanks!

**Comment by rohan-deshpande** [comment, marked as answer]
Heya, thanks for this. I'm not seeing an issue in the codesandbox on Safari on iOS v14.3, the background isn't white but maybe it's because it's not transparent? 

![image](https://user-images.githubusercontent.com/1079637/106658972-2c05ca00-65f2-11eb-9ba5-2a07c9291cce.png)

Can you post a sandbox showing the exact issue? 

Also... is there actually a performance degradation happening here as in the app is rendering slowly or is it just a graphical glitch?

**Comment by rohan-deshpande** [comment]
Locking due to inactivity
